force-app/main/default/classes/PatientRelatedCases.cls

```diff
@@ -5,14 +5,15 @@ public without sharing class PatientRelatedCases {
         Case record = [select ContactId from Case where Id = :caseId];
 
         return [
-            select 
+            Select 
                 Id, CaseNumber, Subject, Status, ClosedDate, Drug_Name__c,
                 Contact.Name, Provider__r.Name, Owner.Name
-            from Case
-            where ContactId = :record.ContactId
-            and Id != :caseId
-            order by CreatedDate asc
+            From Case
+            Where ContactId = :record.ContactId
+            AND Id != :caseId 
+            AND Status IN ('Approved', 'Not Approved', 'More Information Requested', 'Duplicate', 'Currently Approved', 'Cancelled')
+            order BY ClosedDate desc
```

Comment:
@deepakmulamalla There is no uniform with the SOQL statement. Select is in camel case, but AND is in upper-case. If you missed it, please make it uniform across.

Reply:
The SOSL has been changed to upper-case as suggested.

Reply:
@cgiSunilM  Updated the query to all upper-case